gradio/utils.py

```diff
@@ -195,6 +195,17 @@ def kaggle_check() -> bool:
     )
 
 
+def sagemaker_check() -> bool:
+    try:
+        import boto3  # type: ignore
```

Comment:
Is boto3 installed automatically on sagemaker? Wondering if we should tell users that they may need to install boto3 for us to detect if they are running in sagemaker

Reply:
Yes it's installed automatically